discovery_engine_core/web-api/openapi.yaml

```
@@ -368,7 +372,9 @@ components:
         properties:
           $ref: '#/components/schemas/DocumentProperties'
     UserId:
-      description: A user id can be any non-empty UTF-8 string that does not contain the null byte.
+      description: |-
+        A user id can be any non-empty UTF-8 string that does not contain the null byte.
+        It may consist of digits, Latin letters, underscores, colons, minus signs, at signs, and dots.
       type: string
       example: "user id"
```

🔧 space is not valid anymore
```suggestion
      example: "user_id"
```